rpc/handlers_query.go

```diff
@@ -0,0 +1,854 @@
+package rpc
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+	"github.com/pokt-network/pocket/shared/codec"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/utils"
+)
+
+// Queries
+
+func (s *rpcServer) PostV1QueryAccount(ctx echo.Context) error {
+	var body QueryAddressHeight
+	if err := ctx.Bind(&body); err != nil {
+		return ctx.String(http.StatusBadRequest, "bad request")
+	}
+
+	// Get latest stored block height
+	height := body.Height
+	if height == 0 {
+		currentHeight := int64(s.GetBus().GetConsensusModule().CurrentHeight())
+		if currentHeight > 0 {
+			currentHeight -= 1
+		}
+		height = currentHeight
```

Comment:
I am doing this because for the majority of the queries - except a few we want to query the latest committed block (I am presuming here). Consensus height is one above the latest committed block. If this is not the intended I will change. LMK

Reply:
Got it. Can you just add a small helper called `getQueryHeight` which captures this business logic and add a comment explaining what you just explained to me above.

```go
	height := body.Height
	if height == 0 {
		currentHeight := int64(s.GetBus().GetConsensusModule().CurrentHeight())
		if currentHeight > 0 {
			currentHeight -= 1
		}
		height = currentHeight
	}
```